libs/blocks/merch/merch.js

```diff
@@ -629,7 +646,7 @@ export async function getCheckoutContext(el, params) {
 
   return {
     ...context,
-    checkoutClientId,
+    'checkout-client-id': checkoutClientId,
```

Comment:
i'm not sure why we are doing this: mapping in & out from kebab case is done for having well formed attributes but before and after camel case is used iirc

Reply:
true, I have removed it.